Makefile

```diff
@@ -5,16 +5,16 @@ style:
 	black --preview $(check_dirs)
 	isort $(check_dirs)
 
-BATCH_SIZE ?= 1
-DTYPE ?= float16
+BATCH_SIZE ?= 4
```

Comment:
Please don't change the defaults

Reply:
yeah, I was experimenting, sorry will revert this one